lib/MusicBrainz/Server/Entity/Editor.pm

```diff
@@ -324,6 +328,7 @@ sub TO_JSON {
         gravatar => $self->gravatar,
         id => $self->id,
         name => $self->name,
+        privileges => $self->public_privileges,
```

Comment:
Just in case Perl is being stupid:
```suggestion
        privileges => 0 + $self->public_privileges,
```

Reply:
We do just `privileges => $self->privileges` above though. If we change them then I guess both should change?

Reply:
It’s unneeded for `$self->privileges` because the class attribute `privileges` is of type `Int` (line 35). I doubt this is needed here either but did not check if `&` operands and call to subroutine `public_privileges` make any change.

Reply:
The Moose attribute type is just for validation; it doesn't affect how JSON encoder works:

```Perl
#!/usr/bin/env perl

package Editor;
use Moose;
has privileges => ( is => 'ro', isa => 'Int' );
1;

package Main;
use JSON qw( encode_json );
my $x = Editor->new( privileges => '123' );
print encode_json({ privileges => $x->privileges });
1;
```

still outputs `{"privileges":"123"}`, for example.

The determinant factor is whether Perl views the scalar as a number, and you can force that with the `0 +` trick. (You can use a package like `Devel::Peek` to see if there's an "IV" (integer value) item associated with the scalar.)

So yes, it would make sense to add it to `privileges => $self->privileges` too.

Reply:
Added it to both in a second commit.